tests/roles/glance_adoption/tasks/main.yaml

```diff
@@ -1,9 +1,23 @@
 - name: deploy podified Glance with local backend
```

Comment:
Nit: this should now say "with NFS backend"? But we can land the patch and tweak in another one, since the CI is green.